.travis.yml

```diff
@@ -1,18 +1,19 @@
 language: node_js
 node_js:
-  - "8"
-  - "9"
-  - "10"
-  - "12"
+  - 10
```

Comment:
We still need. 8, and 9 since we do still support those versions.

Reply:
updatd 41df984